README.md

```diff
@@ -123,31 +135,42 @@ Configures the general behavior of the application (across all features)
 - Is Mandatory: No (Defaults to INFO)
 
 **TEST_RUN**
+
 - Allows you to safely try out this tool. If active, downloads will not be removed
 - Type: Boolean
 - Permissible Values: True, False
 - Is Mandatory: No (Defaults to False)
 
 **SSL_VERIFICATION**
+
 - Turns SSL certificate verification on or off for all API calls
 - `True` means that the SSL certificate verification is on
-- Warning: It's important to note that disabling SSL verification can have security implications, as it makes the system vulnerable to man-in-the-middle attacks. It should only be done in a controlled and secure environment where the risks are well understood and mitigated 
+- Warning: It's important to note that disabling SSL verification can have security implications, as it makes the system vulnerable to man-in-the-middle attacks. It should only be done in a controlled and secure environment where the risks are well understood and mitigated
 - Type: Boolean
 - Permissible Values: True, False
 - Is Mandatory: No (Defaults to True)
 
+**TIME_ZONE**
```

Comment:
As mentioned in the other comments - I suggest not to include that (and stick with the default TZ, which don't think explanation in my opinion), and rather to include a variable based on which we turn on or off the inclusion of timestamps in logs.